tests/examples/Globl.php

```diff
@@ -37,14 +36,14 @@ public function main()
         $var_f32_type = Type\GlobalType::new(Type\ValType::new(Type\ValType::KIND_F32), Type\GlobalType::MUTABILITY_VAR);
         $var_i64_type = Type\GlobalType::new(Type\ValType::new(Type\ValType::KIND_I64), Type\GlobalType::MUTABILITY_VAR);
 
-        $val_f32_1 = Module\Val::newF32((float) 1);
-        $const_f32_import = Module\Globl::new($store, $const_f32_type, $val_f32_1);
-        $val_i64_2 = Module\Val::newI64(2);
-        $const_i64_import = Module\Globl::new($store, $const_i64_type, $val_i64_2);
-        $val_f32_3 = Module\Val::newF32((float) 3);
-        $var_f32_import = Module\Globl::new($store, $var_f32_type, $val_f32_3);
-        $val_i64_4 = Module\Val::newI64(4);
-        $var_i64_import = Module\Globl::new($store, $var_i64_type, $val_i64_4);
+        $val_f32_1 = Wasm\Val::newF32((float) 1);
+        $const_f32_import = Wasm\Globl::new($store, $const_f32_type, $val_f32_1);
```

Comment:
Is this a typo? Should be Global?

Reply:
as discussed on Slack, `global` (case-insensitive) is a [PHP reserved keyword](https://www.php.net/manual/en/reserved.keywords.php). We can't use it in any context.

Marking the discussion as resolved :)